app/src/main/res/layout-sw600dp/activity_oscilloscope.xml

```diff
@@ -1,291 +1,339 @@
+                    android:background="@color/black"
+                    android:orientation="vertical">
+
+                    <TextView
+                        android:id="@+id/tv_graph_label_xaxis_os"
+                        android:layout_width="wrap_content"
+                        android:layout_height="wrap_content"
+                        android:layout_alignParentTop="true"
+                        android:layout_centerHorizontal="true"
+                        android:layout_gravity="center_vertical|center_horizontal|center"
+                        android:foregroundGravity="center_vertical"
+                        android:gravity="center_vertical|center_horizontal|center"
+                        android:text="@string/time_space"
+                        android:textColor="@color/white"
+                        android:textSize="12sp"
+                        android:textStyle="normal|bold"
+                        tools:layout_editor_absoluteX="288dp"
+                        tools:layout_editor_absoluteY="0dp" />
+
+                    <TextView
+                        android:id="@+id/tv_unit_xaxis_os"
+                        android:layout_width="wrap_content"
+                        android:layout_height="wrap_content"
+                        android:layout_alignParentTop="true"
+                        android:layout_marginStart="8dp"
+                        android:layout_marginLeft="8dp"
+                        android:layout_toEndOf="@+id/tv_graph_label_xaxis_os"
+                        android:layout_toRightOf="@+id/tv_graph_label_xaxis_os"
+                        android:text="@string/unit_μs"
+                        android:textColor="@color/white"
+                        android:textSize="12sp"
+                        android:textStyle="normal|bold"
+                        app:layout_constraintLeft_toRightOf="@+id/tv_graph_label_xaxis_os"
+                        tools:layout_editor_absoluteY="0dp" />
+                </RelativeLayout>
+
+
+                <android.support.constraint.ConstraintLayout
+                    android:id="@+id/chart_yaxis_layout1"
+                    android:layout_width="wrap_content"
+                    android:layout_height="match_parent"
+                    android:layout_alignParentStart="true"
+                    android:layout_alignParentLeft="true"
+                    android:layout_alignParentTop="true"
+                    android:background="@color/black"
+                    android:foregroundGravity="center_vertical"
+                    android:gravity="center_vertical"
+                    android:orientation="vertical">
+
+                    <TextView
+                        android:id="@+id/tv_label_left_yaxis_os"
+                        android:layout_width="wrap_content"
+                        android:layout_height="wrap_content"
+                        android:layout_gravity="center"
+                        android:layout_marginTop="16dp"
```

Comment:
Hello, I see couple of dimensions are hard coded. Shall we add them to dimens.xml and refer from there?

Reply:
👍 I'll have a look and I'll update the pull request when I am done.

Reply:
The whole layout file was more or less an outdated and incomplete copy of the general oscilloscope layout file. I deleted the file and the Activity still renders fine. Deleting the layout file also removes redundancy which is always nice.